smarts/core/mission_planner.py

```diff
@@ -228,7 +229,7 @@ def cut_in_waypoints(self, sim, pose: Pose, vehicle):
         target_offset = self._road_network.offset_into_lane(
             target_lane, target_position
         )
-        if offset < target_offset:
+        if offset < target_offset + 5:
```

Comment:
In theory this padding should also be parameterizable. It might be useful calculate the padding using some value related to the road speed or vehicle size, or parameterization from the task.

Reply:
Hmm, the only reason not doing it now is because if it's parameterizable then we should make sure the input from users is valid, reasonable, and practical.

Reply:
:ok_hand: